.gitignore

```diff
@@ -1,3 +1,4 @@
 agent/src/bpf/vmlinux.h
+agent/tests/agenttest/src/bpf/vmlinux.h
```

Comment:
why?

Reply:
Because it is generated on the fly at `cargo build` using bpftool.